src/gui/addnewtorrentdialog.cpp

```diff
@@ -79,8 +79,8 @@ namespace
     }
 }
 
-constexpr int AddNewTorrentDialog::minPathHistoryLength;
-constexpr int AddNewTorrentDialog::maxPathHistoryLength;
+const int AddNewTorrentDialog::minPathHistoryLength;
```

Comment:
Why these changes?

Reply:
For me, constexpr isn't really required here and it makes the code a bit harder to read.